docs/providers/aws/cli-reference/config.md

```diff
@@ -0,0 +1,50 @@
+<!--
+title: Serverless Framework Commands - AWS Lambda - Config
+menuText: Config
+menuOrder: 10
+description: Configure Serverless.
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/providers/aws/cli-reference/config)
+<!-- DOCS-SITE-LINK:END -->
+
+# Config
+
+This plugin helps you to configure Serverless.
+
+## Configure credentials
+
+```bash
+serverless config credentials --provider provider --key key --secret secret
+```
+
+### Options
+
+- `--provider` or `-p` The provider (in this case `aws`). **Required**.
+- `--key` or `-k` The `aws_access_key_id`. **Required**.
+- `--secret` or `-s` The `aws_secret_access_key`. **Required**.
+- `--profile` or `-n` The name of the profile which should be created.
+
+### Provided lifecycle events
+
+- `config:credentials:credentials`
```

Comment:
@pmuens just checked other functions and there we use `'invoke:local:invoke'` and `'deploy:function:deploy'`. You definitely have my vote to go for `'config:credentials:config'` since it would be consistent with our other decisions.